pkg/plugins/backendplugin/secretsmanagerplugin/buf.gen.yaml

```
@@ -0,0 +1,10 @@
+version: v1
+plugins:
+  - plugin: go
+    out: pkg/plugins/backendplugin/secretsmanagerplugin
+    opt: paths=source_relative
+  - plugin: go-grpc
+    out: pkg/plugins/backendplugin/secretsmanagerplugin
+    opt:
+      - paths=source_relative
+      - require_unimplemented_servers=false
```

Looks like the existing grpc is generated with this enabled?
```suggestion
      - require_unimplemented_servers=true
```